README.md

```diff
@@ -1 +1,55 @@
-# create-nuget-package-action
\ No newline at end of file
+# create-nuget-package-action
+
+<img alt= "" src="https://github.com/OctopusDeploy/create-nuget-package-action/raw/main/assets/github-actions-octopus.png" />
+
+This is a GitHub Action to create a NuGet package to push [Octopus Deploy](https://octopus.com/).
+
+## Examples
+
+Incorporate the following actions in your workflow to push a package to Octopus Deploy using an API key, a target instance (i.e. `server`), and a project:
```

Comment:
We didn't read this earlier, doesn't make sense for these actions (suspect this is wrong for the Zip one too, I'll fix it there.